Android/Kotlin/app/src/main/java/com/ironsource/ironsourcesdkdemo/DemoActivity.kt

```diff
@@ -11,29 +11,39 @@ import android.widget.Button
 import android.widget.FrameLayout
 import android.widget.TextView
 import androidx.multidex.BuildConfig
-import com.ironsource.mediationsdk.ISBannerSize
 import com.ironsource.mediationsdk.IronSource
-import com.ironsource.mediationsdk.IronSourceBannerLayout
 import com.ironsource.mediationsdk.integration.IntegrationHelper
 import com.ironsource.mediationsdk.model.Placement
 import com.ironsource.mediationsdk.utils.IronSourceUtils
+import com.unity3d.mediation.LevelPlay
+import com.unity3d.mediation.LevelPlayAdSize
+import com.unity3d.mediation.LevelPlayInitRequest
+import com.unity3d.mediation.banner.LevelPlayBannerAdView
+import com.unity3d.mediation.interstitial.LevelPlayInterstitialAd
 
 
 private const val TAG = "DemoActivity"
+
+// Replace with your app key as available in the LevelPlay dashboard
 private const val APP_KEY = "85460dcd"
 
+// Replace with your ad unit ids as available in the LevelPlay dashboard
+private const val INTERSTITIAL_AD_UNIT_ID = "aeyqi3vqlv6o8sh9"
+private const val BANNER_AD_UNIT_ID = "thnfvcsog13bhn08"
+
 class DemoActivity : Activity(), DemoActivityListener {
 
     private lateinit var rewardedVideoShowButton: Button
     private lateinit var interstitialLoadButton: Button
     private lateinit var interstitialShowButton: Button
     private lateinit var bannerLoadButton: Button
-    private lateinit var bannerDestroyButton: Button
 
     private var bannerParentLayout: FrameLayout? = null
-    private var ironSourceBannerLayout: IronSourceBannerLayout? = null
     private var rewardedVideoPlacementInfo: Placement? = null
 
+    private var mInterstitialAd : LevelPlayInterstitialAd? = null
+    private var mBannerAd : LevelPlayBannerAdView? = null
```

Comment:
please divide the properties per adFormat. 
And as I see, the other properties don't include "m" prefix. I would align to one approach.